src/org/javarosa/core/model/actions/SetValueAction.java

```diff
@@ -78,7 +78,9 @@ public void processAction(FormDef model, TreeReference contextRef) {
 		}
 
 		AbstractTreeElement node = context.resolveReference(qualifiedReference);
-		if(node == null) { throw new NullPointerException("Target of TreeReference " + qualifiedReference.toString(true) +" could not be resolved!"); }
+		if(node == null) {
```

Comment:
I've fixed only LeftCurly checkstyle. A space is another one and I can fix it in the future as a separate pr.